ts/src/bigone.ts

```diff
@@ -173,6 +380,190 @@ export default class bigone extends Exchange {
         });
     }
 
+    async fetchCurrencies (params = {}) {
+        /**
+         * @method
+         * @name bigone#fetchCurrencies
+         * @description fetches all available currencies on an exchange
+         * @param {dict} params extra parameters specific to the aax api endpoint
+         * @returns {dict} an associative dictionary of currencies
+         */
+        // we use undocumented link (possible, less informative alternative is : https://big.one/api/uc/v3/assets/accounts)
+        const data = await this.fetchWebEndpoint ('fetchCurrencies', 'webExchangeGetUcV2Assets', true);
+        if (data === undefined) {
+            return undefined;
```

Comment:
@ttodua if the user disabled web fetching, shouldn't we be using this endpoint instead? https://big.one/api/uc/v3/assets/accounts

Reply:
that endpoint is also web-endpoint exactly like the one being used now. if user has disabled web-fetching, then it should also become disabled.

Reply:
@ttodua oh ok nvm then I was assuming it was a proper api endpoint